tests/bot/cogs/moderation/test_utils.py

```diff
@@ -77,54 +77,76 @@ def setUp(self):
         test_cases = [
             {
                 "args": (self.user, "ban", "2020-02-26 09:20 (23 hours and 59 minutes)"),
-                "expected_output": {
-                    "description": INFRACTION_DESCRIPTION_TEMPLATE.format(
+                "expected_output": Embed(
+                    title=INFRACTION_TITLE,
+                    description=INFRACTION_DESCRIPTION_TEMPLATE.format(
                         type="Ban",
                         expires="2020-02-26 09:20 (23 hours and 59 minutes)",
                         reason="No reason provided."
                     ),
-                    "icon_url": Icons.token_removed,
-                    "footer": INFRACTION_APPEAL_FOOTER,
-                },
+                    colour=INFRACTION_COLOR,
+                    url=utils.RULES_URL
+                ).set_author(
+                    name=INFRACTION_AUTHOR_NAME,
+                    url=utils.RULES_URL,
+                    icon_url=Icons.token_removed
+                ).set_footer(text=INFRACTION_APPEAL_FOOTER),
```

Comment:
Ultimately I'm not sure if this turned out better than what existed before. I'll let the other reviewer(s) have the final say and I'm just gonna accept this as it is.